src/buttons/button-icon.ts

```diff
@@ -18,4 +19,9 @@ export class NglButtonIcon {
     this.renderer.setElementClass(this.element.nativeElement, 'slds-button', true);
   }
 
-};
+  private normalize(type: string): string {
+    if (type == null) return '-border';
```

Comment:
@esarbanis Can you just `if` it instead of comparing it with `null`. I suppose we want to handle all cases like `null`, `undefined`, `false` or empty string the same way. Is it correct?